docs/source/developer_guide/development-workflow.md

```diff
@@ -59,12 +60,30 @@ Download and install a [Python 3 version of Miniconda](https://docs.conda.io/en/
     ```
     conda install -y -c conda-forge/label/main nodejs
     ```
+* Verify node is installed correctly 
+
+    ```
+    node --version 
+    ```
 
 * Install Yarn
 
     ```
     conda install -y -c conda-forge/label/main yarn
     ```
+* Verify yarn is installed correctly 
+
+    ```
+    yarn --version 
+    ```
+
+* Install GNU Make 
+
+    Refer to the following link for installation instructions: 
+    [GNU Make](https://www.gnu.org/software/make/)
+
+    To verify the installation, run `make`. 
+    Check with the output written below in [Build and Installation](#build-installation) to confirm that you have correctly installed `make`.
```

Comment:
I think this may be confusing if they have yet to set up the repository, which is the next step.  If there is no `Makefile` present, they will receive the message: `make: *** No targets specified and no makefile found.  Stop.`

Perhaps indicating both scenarios might be better, also making sure that `make` is invoked in the repository directory as well.

```suggestion
    To verify the installation, run `make`. 
    If you have yet to [set up the repository](#setting-up-your-development-environment), you should see a message like the following:
    ```
    make: *** No targets specified and no makefile found.  Stop.
    ```
    Once the repository is set up, running `make` from that location should display the available tasks that are listed in the [Build & Installation](#build-installation) section below.
```